docs/openwrt_compile_source/compile_tinyHIPPO_from_source.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Uninstall the old cids package
+echo 'Uninstalling old tinyHIPPO package'
+$HOME/source/scripts/feeds uninstall tinyHIPPO
```

Comment:
Will this fail or throw an error if the user has never installed tinyHIPPO before?

Reply:
If the tinyHIPPO package does not exist, it will just print 'Could not uninstall tinyHIPPO' to STDOUT. It does not affect the functionality of the script.